registration-processor/core-processor/registration-processor-manual-verification-stage/src/main/java/io/mosip/registration/processor/manual/verification/service/impl/ManualVerificationServiceImpl.java

```diff
@@ -155,6 +165,10 @@
 
 	@Value("${mosip.regproc.data.share.internal.domain.name}")
 	private String internalDomainName;
+	
+	
+	@Value("${mosip.regproc.use.manual.adjudication.request.old}")
```

Comment:
can you please change it to
mosip.regproc.manual.adjudication.use.lts.format=true

Reply:
I have changed the name